assignments/array-methods.js

```diff
@@ -56,28 +56,64 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+let newfullName = runners.forEach(function(i){
+    fullName.push(i.first_name + " " + i.last_name);
```

Comment:
We are using forEach to affect each element of the array, and then pushing something to an array we already called. We don't need to save the results in `newfullName`.